app.config.js

```diff
@@ -48,4 +42,20 @@ const extraConfig = {
 }
 configurationFiles.push(extraConfig)
 
+if (
+  isPhotos &&
+  configurationFiles[0].multiple &&
+  configurationFiles[0].multiple.services
+) {
+  // FIXME: Will be handled correctly by next major version of cozy-scripts
+  configurationFiles[0].multiple.services.__mergeStrategy.strategy[
+    'resolve.modules'
+  ] = 'replace'
+  configurationFiles[0].multiple.services.resolve.modules = [
+    SRC_DIR,
+    'node_modules'
+  ]
```

Comment:
This fails if resolve is not defined.
```suggestion
      configurationFiles[0].multiple.services.resolve = {
    ...configurationFiles[0].multiple.services.resolve,
    modules: [SRC_DIR, 'node_modules']
  }
```